content/en/agent/autodiscovery/_index.md

```diff
@@ -129,19 +115,20 @@ ECS_FARGATE=true
 {{% /tab %}}
 {{< /tabs >}}
 
+### Integration Templates
```

Comment:
Can you add an example and an explanation to it?

Reply:
added quick redis example below, users should follow link to Integrations Templates page to read more.